plugins/by-name/otter/default.nix

```diff
@@ -113,6 +113,10 @@ lib.nixvim.plugins.mkNeovimPlugin {
     plugins.lsp.onAttach = lib.mkIf cfg.autoActivate ''
       require('otter').activate()
     '';
+
+    lsp.onAttach = lib.mkIf cfg.autoActivate ''
+      require('otter').activate()
+    '';
```

Comment:
Should we do this _instead_ of `plugins.lsp.onAttach`?

IIRC, `lsp.onAttach` is always used (if non-empty), while `plugins.lsp.onAttah` is only used if `plugins.lsp.enable`.

In effect, this means that if you have a line in both `onAttach` options, it'll run once when `plugins.lsp.enable = false` and **twice** when `plugins.lsp.enable = true`.

Unless I'm forgetting something?

Reply:
Migrated to the new lsp module instead of using both